exporter/exporter.go

```diff
@@ -509,6 +509,7 @@ func NewRedisExporter(uri string, opts Options) (*Exporter, error) {
 
 	e.mux.HandleFunc("/", e.indexHandler)
 	e.mux.HandleFunc("/scrape", e.scrapeHandler)
+	e.mux.HandleFunc("/discovery", e.discoveryHandler)
```

Comment:
Oh, one more thing, can you please call this `/discover-cluster-nodes` ? 
It's a bit longer but makes it more clear that this is for discovering cluster nodes.